### PR TITLE
Fixes #180: Merge forest level into the game

### LINKED ISSUE #180: Merge forest level into the game
### Expected results

We can play the forest level by choosing from a list of levels.

### Actual results

We have a different file and a lot of duplicated code:

- https://github.com/fossasia/labyrinth/blob/master/js/tilesForest.js
- https://github.com/fossasia/labyrinth/blob/master/js/levelsForest.js

### Hints:

To solve this issue, we can do two basic operations:

- in tilesForest.js:
  1. copy the code to the tiles.js file
  2. rename "doors:" to forest
- in levelsForest.js:
  1. copy the level over to levels.js
  2. rename the function to createForestLevel
  3. rename all "doors" tiles in the forest level to "forest"

This is a refactoring issue to improve the code quality.

After this, #161 should take care of the rest.

### PR DESCRIPTION
**Pull request by abishekvashok**
This pull request is related to issue #180 

The problem I want to solve is: #180 

My steps to solve it:
Changes:
Merges forest level into the game by adding the level definition to
levels.js which will be rectified in #161, tiles implementation under
forest class to tiles.js and deleted the then reduntant files.

Fixes #180

Screenshots/Links: -- Not applicable

### CODE REVIEW

**Comment by samyakbvs** [comment]
How will the player go from one level to another @abishekvashok

**Comment by abishekvashok** [comment]
@samyakbvs it will be ready after #161 is finished. We will return a new level and start the procedure from start to bottom on reach of the goal function. I, @0x48piraj and @YashKumarVerma are working on it :)

File: js/levels.js [hunk 1]
```diff
@@ -33,4 +32,15 @@ function createTestLevel()
     [door.left, door.top, door.none, door.right, door.none, door.wheel, door.none, door.none, door.none],
     [NullTile, door.none, door.none, door.none, door.none, door.none, door.left, door.top, door.right, door.top]
 ]);
+  /* I know this return doesnot work but leaving it here to be solved in #161 */
+  /* Forest specific levels */
+  return new Level([
```
Comment (niccokunzmann) [inline js/levels.js]: I think, here we can use a new function body. This code is not reachable, right? I guess so.

Comment (abishekvashok) [reply js/levels.js]: We would be fixing this in #160

Comment (abishekvashok) [reply js/levels.js]: Sorry #161

**Pull request merged**